### DISCUSSION TITLE
Less renders

### DISCUSSION THREAD

**Discussion by dlamb22** [apenab/react-dymo#31] [General]
Is there any way you can get this to render once instead of 4 times?

**Comment by apenab** [comment]
Hi. I have to make 4 requests to guess the "web service port" then I save in the local storage the "web service port" and no more re-renders. If you see more unnecessary renders, plz, give me more details

**Reply by apenab** [reply]
@dlamb22

**Comment by dlamb22** [comment]
Thank you. I did not see in the docs about saving the info in local storage.

**Comment by apenab** [comment]
This was fixed in the latest release
